pkgs/tools/misc/SP800-90B_EntropyAssessment/default.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, bzip2
+, libdivsufsort
+, jsoncpp
+, openssl
+, mpfr
+}:
+
+stdenv.mkDerivation rec {
+  pname = "SP800-90B_EntropyAssessment";
+  version = "1.1.6";
+
+  src = fetchFromGitHub {
+    owner = "usnistgov";
+    repo = "SP800-90B_EntropyAssessment";
+    rev = "v${version}";
+    hash = "sha256-KZQ7kC0PbBkjLEQZIqYakQ91OvCxruhdfUwiRHtno3w=";
+  };
+
+  buildInputs = [ bzip2 libdivsufsort jsoncpp openssl mpfr ];
+
+  sourceRoot = "source/cpp";
+
+  makeFlags = lib.optionals (stdenv.buildPlatform != stdenv.hostPlatform) [
```

```suggestion
  makeFlags = [
```

Looked at the makefile and it should be ok to have these unconditional

---

It is changed now, like you suggested. Also the builder for aarch64 succeeded now.